source/locale/fr/LC_MESSAGES/presentation.po

```diff
@@ -39,8 +39,8 @@ msgid ""
 "git-flow propose. It is just simplier to use, and it prevents to use the "
 "incorect branch, or to forget about merging somewhere."
 msgstr ""
-"Gardez en tête qu'il s'agit de standardiser ; des commandes git standard sont "
-"utilisées an arrière-plan, vous pourriez obtenir le même résultat « "
+"Gardez en tête qu'il s'agit de standardiser ; des commandes git standardes sont "
```

Comment:
```suggestion
"Gardez en tête qu'il s'agit de standardiser ; des commandes git standard sont "
```

Reply:
'standard' is an adjective: we must accord it in gender and number.

Reply:
«Employé comme adjectif, standard ne varie qu'en nombre : des prix, des formules standards. » => https://www.larousse.fr/dictionnaires/francais/standard/74453

"standarde" does not exists.

Reply:
OK, I revert this one.